keyboards/karlb/kbic65/rules.mk

```
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

We do recommend enabling bootmagic lite by default.  Especially, as it's enabled as part of VIA.
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```